examples/temporal/temporal.js

```diff
@@ -0,0 +1,54 @@
+( function () {
```

Comment:
this file should be upper case it describe a class, if moving it to osgjs source tree it would be renamed. It's good to keep the same rule in examples